pkg/api/defaults-kubelet.go

```diff
@@ -199,6 +201,12 @@ func removeKubeletFlags(k map[string]string, v string) {
 			delete(k, key)
 		}
 	}
+
+	if strings.ToLower(osType) == "windows" {
+		for _, key := range []string{"--tls-cert-file", "--tls-private-key-file"} {
```

Comment:
instead of doing this, you could also add these values as empty `""` to `staticWindowsKubeletConfig ` like we do for  `--pod-manifest-path` and they would get deleted by https://github.com/Azure/aks-engine/pull/1416/files#diff-ac6b80e7a914ca77b31a38a10c77826bR198

Reply:
that's a good suggestion, it would make the code logic more clear, I have followed your guide, thanks, PTAL.